src/app/bungie-api/authenticated-fetch.ts

```diff
@@ -74,18 +74,21 @@ async function responseIndicatesBadToken(response: Response) {
   if (response.status === 401) {
     return true;
   }
-  const data = await response.clone().json();
-  return (
-    data &&
-    (data.ErrorCode === PlatformErrorCodes.AccessTokenHasExpired ||
-      data.ErrorCode === PlatformErrorCodes.WebAuthRequired ||
-      // (also means the access token has expired)
-      data.ErrorCode === PlatformErrorCodes.WebAuthModuleAsyncFailed ||
-      data.ErrorCode === PlatformErrorCodes.AuthorizationRecordRevoked ||
-      data.ErrorCode === PlatformErrorCodes.AuthorizationRecordExpired ||
-      data.ErrorCode === PlatformErrorCodes.AuthorizationCodeStale ||
-      data.ErrorCode === PlatformErrorCodes.AuthorizationCodeInvalid)
-  );
+  try {
+    const data = await response.clone().json();
```

Comment:
LMAO i just finished adding the same try in a different copy of this
![image](https://user-images.githubusercontent.com/68782081/144582604-99099771-4f33-4133-874f-57be90826721.png)